From f1d5cf2b564fa9b1c613d940bad9988d134e41e1 Mon Sep 17 00:00:00 2001
From: Jason Zaugg <jzaugg@gmail.com>
Date: Tue, 20 Oct 2026 00:34:27 +1000
Subject: [PATCH] Integration: rz#63's merkle-discovery-overridden test opts
 into hashing=merkle

rz#83 makes Merkle hashing opt-in; rz#63's new scripted test predates it.
---
 .../merkle-discovery-overridden/incOptions.properties            | 1 +
 1 file changed, 1 insertion(+)

diff --git a/zinc/src/sbt-test/source-dependencies/merkle-discovery-overridden/incOptions.properties b/zinc/src/sbt-test/source-dependencies/merkle-discovery-overridden/incOptions.properties
index 2673f7863..f0d13f3c0 100644
--- a/zinc/src/sbt-test/source-dependencies/merkle-discovery-overridden/incOptions.properties
+++ b/zinc/src/sbt-test/source-dependencies/merkle-discovery-overridden/incOptions.properties
@@ -1 +1,2 @@
+hashing = merkle
 apiDebug = false
-- 
2.56.0

